app/services/shopify/association_saver.rb

```diff
@@ -0,0 +1,24 @@
+module Shopify
```

Comment:
Cool! So in this branch this has been moved out to it's own file? I guess you can ignore the other comment in the other branch.